docs/candy-machine-v2/02-configuration.md

```diff
@@ -83,6 +83,10 @@ The `number` of items setting can only be updated after you create your `CMv2` i
 In case you require to change the `number` of items after creating a `CMv2` without `hiddenSettings`, you can withdraw rent of your current `CMv2` and then create a new one.
 :::
 
+:::info
+The `storage` must be set to `arweave-bundle` or `arweave-sol` for mainnet and `arweave`, `aws` or `ipfs` for devnet.
```

Comment:
The current version of the text suggests that `arweave-bundle` and `arweave-sol` are the only options for mainnet, which is not accurate. We should clarify that these option are only available on mainnet (not devnet).

Reply:
Hi, 
>  We should clarify that these options are only available on mainnet (not devnet).
Isn't this what the following comment states:
> must be set to `arweave-bundle` or `arweave-sol` for mainnet 

Maybe there is some misinterpretation here.

Reply:
When you say that `arweave-bundle` or `arweave-sol` "must be set for mainnet", it implies that these are the only options for mainnet. This is not the case, they are options only available for mainnet. I suggest the following text:

"The `storage` options `arweave-bundle` and `arweave-sol` are only available on mainnet."

On the table just above, it is mentioned that these options are only available on mainnet, so I wonder if we should emphasise the text there instead of adding another "info" box.

Reply:
You are right about the already present info in the table. Just added a patch to the PR. I think this fits better, please have a look.